File: app/src/release/java/com/stedi/randomimagegenerator/app/other/logger/NullLogger.java
package com.stedi.randomimagegenerator.app.other.logger;

import android.support.annotation.NonNull;
import android.support.annotation.Nullable;

public class NullLogger implements Logger {
    @Override
    public void log(@NonNull Object from, @Nullable String message) {

    }

    @Override
    public void log(@NonNull Object from, @Nullable Throwable t) {

    }

    @Override
    public void log(@NonNull Object from, @Nullable String message, @Nullable Throwable t) {

    }
}

File: app/src/release/java/com/stedi/randomimagegenerator/app/other/logger/NullLogger.kt
package com.stedi.randomimagegenerator.app.other.logger

class NullLogger : Logger {
    override fun log(from: Any, message: String?) {
    }

    override fun log(from: Any, t: Throwable?) {
    }

    override fun log(from: Any, message: String?, t: Throwable?) {
    }
}